data/ideas.json

```diff
@@ -40,5 +40,11 @@
         "description": "Create a rock paper scissors game! Try recreating the classic game, or put your own fun twist on it.",
         "level": "basic",
         "tags": "rock-paper-scissors, game, classic"
+    },
+    {
+        "title": "A personal assistant for your system using python",
```

Comment:
Could you remove the programming language name? due to this could be programming on another language

Reply:
Thanks for reviewing, and surely I will do all the changes as required.